io.openems.edge.evcs.cluster/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="src" output="bin" path="src"/>
+	<classpathentry kind="src" path="/io.openems.edge.core"/>
```

Comment:
This should not be in the classpath. Maybe you clicked something wrong in Eclipse. Also the 'test' folder is missing here. Compare with other classpath files of other projects - they are always identical.